.ci_support/linux_64_.yaml

```diff
@@ -1,10 +1,8 @@
-cdt_name:
-- conda
 channel_sources:
 - conda-forge
 channel_targets:
 - conda-forge main
 docker_image:
 - quay.io/condaforge/linux-anvil-x86_64:alma9
 python_min:
-- '3.9'
+- '3.10'
```

Comment:
Does that mean we would up the minimum python requirement to 3.10? Why is that needed?